src/js/modules/infragistics.ui.combo.js

```diff
@@ -1965,9 +1965,31 @@
 			if ($.ig && $.ig.formatter) {
 				if (this.options.format === "auto" &&
 					($.type(item) === "date" || $.type(item) === "number")) {
-					item = $.ig.formatter(item, null, null);
+					item = $.ig.formatter(
```

Comment:
Will the formatter be refactored to take an object of a type? Like `FormatterArgs`?